mct/m_SPMDutils.F90

```diff
@@ -24,6 +24,16 @@
 !
 ! !INTERFACE:
 
+! Code added as a work around for poor performance or incorrect functionality
+! in MPI libraries
+!
+#ifndef _USE_MPI_RSEND
```

Comment:
If we do this in one place, we should do it in the other. I'll add a commit. 

However, how would I set this token when building MCT in ACME?